app/views/settings/preferences/appearance/show.html.haml

```diff
@@ -8,6 +8,10 @@
     .fields-group.fields-row__column.fields-row__column-6
       = f.input :setting_theme, collection: Themes.instance.names, label_method: lambda { |theme| I18n.t("themes.#{theme}", default: theme) }, wrapper: :with_label, include_blank: false, hint: false
 
+  - if I18n.locale.to_s != 'enn'
```

Comment:
```haml
- unless I18n.locale == :en 
```

Reply:
Oh. I used `enn` for testing, but stashed this change, not sure how it end up getting here. And thanks for the tip about `unless`, I already forgot about it since Jekyll. Will fix in a minute.